abstract.tex

```diff
@@ -0,0 +1,14 @@
+\begin{abstract}
+
+The Vera C. Rubin Observatory will revolutionize all areas of astronomy over the next decade with its unique wide-fast-deep multi-color imaging survey, the Legacy Survey of Space and Time (LSST).
```

Comment:
To be fair, LSST might not revolutionize _all_ areas of astronomy. For example, other facilities are much more effective for exoplanet research. Maybe think about an alternative phrasing.

Reply:
many areas of astronomy?